client/core/trade_simnet_test.go

```diff
@@ -618,25 +627,26 @@ HELPER TYPES, FUNCTIONS AND METHODS
 ************************************/
 
 type tWallet struct {
-	name    string
-	account string
-	pass    []byte
-	config  map[string]string
+	daemon     string
+	account    string // for dcr wallets
+	walletName string // for btc wallets
+	pass       []byte
```

Comment:
Noting that #587 was merged today, so the password for all wallets is now `"abc"`.